modules/vault/vault_test.go

```diff
@@ -37,6 +37,7 @@ func TestMain(m *testing.M) {
 		testcontainervault.WithToken(token),
 		// }
 		// WithInitCommand {
+		testcontainervault.WithInitCommand("secrets enable -version=2 kv"),
```

Comment:
After reading [this](https://developer.hashicorp.com/vault/docs/secrets/kv/kv-v2#setup)

Reply:
Doesn’t works...